analytics_data_api/management/commands/generate_fake_course_data.py

```diff
@@ -98,8 +99,9 @@ def generate_daily_data(self, course_id, start_date, end_date):
 
             for mode, ratio in enrollment_mode_ratios.iteritems():
                 count = int(ratio * daily_total)
+                cumulative_count = max(cumulative_count + 10, count)
```

Comment:
My thought was that the cumulative counts will generally go higher as the counts change.  It's not particularly realistic, but does generate the data.